### PR TITLE
WisdomTree Fund (RWA)

### PR DESCRIPTION
**Pull request by 0xpeluche**
---
##### Name (to be shown on DefiLlama):
WisdomTree Government Money Market Digital Fund

##### Website Link:
https://www.wisdomtree.com/investments/digital-funds/money-market/wtgxx
https://www.wisdomtree.com/investments/digital-funds/fixed-income/flttx
https://www.wisdomtree.com/investments/digital-funds/fixed-income/wtsyx
https://www.wisdomtree.com/investments/digital-funds/fixed-income/wttsx
https://www.wisdomtree.com/investments/digital-funds/fixed-income/tipsx
https://www.wisdomtree.com/investments/digital-funds/fixed-income/wtstx
https://www.wisdomtree.com/investments/digital-funds/fixed-income/wtlgx
https://www.wisdomtree.com/investments/digital-funds/equity/spxux
https://www.wisdomtree.com/investments/digital-funds/asset-allocation/wtsix

##### Chain:
- Stellar
- Ethereum

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/wisdomtree exports TVL: 
        
 
 
    ethereum                  11.25 M
    stellar                   679.29 k
    
    total                    11.93 M

**Comment by llamatester** [comment]
The adapter at projects/wisdomtree exports TVL: 
        
 
 
    ethereum                  11.28 M
    stellar                   3.99 M
    
    total                    15.27 M

**Comment by llamatester** [comment]
The adapter at projects/wisdomtree exports TVL: 
        
 
 
    stellar                   12.09 M
    ethereum                  11.27 M
    
    total                    23.36 M

File: projects/wisdomtree/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  ],
+  ethereum: '0x1fecf3d9d4fee7f2c02917a66028a48c6706c179' // WTGX
+}
+
+const stellarTvl = async (api) => {
+  const assets = CONFIG[api.chain]
+  for (const { ticker, address, NAV } of assets) {
+    const stellarApi = `https://api.stellar.expert/explorer/public/asset/${ticker}-${address}`
+    const { data } = await axios.get(stellarApi)
+    api.add(USDT, NAV * data.supply * 10 ** (6-7), { skipChain: true })
```
Comment (g1nt0ki) [inline projects/wisdomtree/index.js:37]: you can do `api.addUSDValue`

File: projects/wisdomtree/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+   { ticker: "WTLG", address: 'GAK7PE7DD4ZRJQN3VBCQFBKFV53JGUM2SQATQAKLFK6MVONPGNYK34XH', NAV: 8.645 }, // WTLGX
+   { ticker: "SPXU", address: 'GDJBVX3QA5HJPBSAU5VIX2W6MC37NU4UFXPKEGK42SJCYN6AEQ4Z6COM', NAV: 15.815 }, // SPXUX
+   { ticker: "WTSI", address: 'GAD22PDBRFEMXAKPFDP4JGDFWKKD6VPXWUWEAXBS6ZYJYFFQDUN7HAFG', NAV: 10.282 }, // WTSIX
+  ],
+  ethereum: '0x1fecf3d9d4fee7f2c02917a66028a48c6706c179' // WTGX
+}
+
+const stellarTvl = async (api) => {
+  const assets = CONFIG[api.chain]
+  for (const { ticker, address, NAV } of assets) {
```
Comment (g1nt0ki) [inline projects/wisdomtree/index.js:34]: hmm, dont like the idea of hardcoding nav

**Comment by llamatester** [comment]
The adapter at projects/franklinTempleton-finance exports TVL: 
        
 
 
    stellar                   267.73 M
    arbitrum                  51.90 M
    polygon                   32.23 M
    avax                      30.06 M
    ethereum                  30.06 M
    base                      30.06 M
    
    total                    442.06 M

**Comment by llamatester** [comment]
The adapter at projects/wisdomtree exports TVL: 
        
 
 
    stellar                   12.09 M
    ethereum                  11.26 M
    
    total                    23.35 M

**Pull request merged**